app/chronology/models.py

```diff
@@ -1,14 +1,20 @@
-from django.contrib.postgres.fields import JSONField
+from typing import Dict
+
 from django.db import models
 
+from common.json_field import JSONField
 from crowd.models import User
 from notice_board.models import Meeting, Sphere
 
 
+def default_festival_settings() -> Dict[str, None]:
+    return {"theme": None, "forms": None}
```

Comment:
Could we use this instead?
 ```suggestion
     return {"theme": {}, "forms": []}
```

Reply:
Anybody can still remove these keys, but they're a bit more "inviting" if we set them to empty dict and list.